Khan/wonder-blocks
# Allow disabled buttons to be focusable by screenreaders

## Summary:
"There might be controls like a button that's disabled and we want to be able to communicate why it's disabled with a tooltip.  We want that tooltip to be accessible even if the control is disabled.  Disabled controls may not tabbable but users can still navigate to them via Voice Over and other screen readers."

- Change `disabled` prop to `aria-disabled`
- Change tabIndex to 0
- We don't have to add an eventListener to stop clicks from happening because that is already handled here:
  https://github.com/Khan/wonder-blocks/blob/master/packages/wonder-blocks-clickable/src/components/clickable-behavior.js#L213

Issue: https://khanacademy.atlassian.net/browse/WB-393

Screen reader:

https://user-images.githubusercontent.com/13231763/146093822-5fc4a327-2228-43a3-b2dc-f291c6b90cf9.mov

Just tabbing:

https://user-images.githubusercontent.com/13231763/148317084-3bf55d9d-0f87-4885-8e45-e50430bf61b6.mov

With tooltips:

https://user-images.githubusercontent.com/13231763/148317089-58cd708e-1548-4200-a771-e7e182fc94b9.mov




## Test plan:
- Open storybook and go to the Basic Buttons tab
- Tab through the buttons. They should all receive focus
  (but you won't be able to see it for the disabled buttons)
- Turn on a screen reader
- Tab through the buttons. Confirm that the screenreader
  reads the disabled buttons as "dimmed"
